adage/analyze/tests.py

```diff
@@ -1025,3 +1034,107 @@ def test_expressionvalue_big_post(self):
+                'gets adapted, P. aeruginosa can persist for several decades '
+                'in the respiratory tracts of CF patients, overcoming host '
+                'defense mechanisms as well as intensive antibiotic '
+                'therapies. P. aeruginosa CF strains isolated from different '
+                'infection stage were selected for RNA extraction and '
+                'hybridization on Affymetrix microarrays. Two batch of P. '
+                'aeruginosa CF isolates are chosen : 1) isolates from a '
+                'group of patients since 1973-2008 as described in ref '
+                '(PMID: 21518885); 2) isolates from a group of newly '
+                'infected children as described in ref (PMID: 20406284).'
+        }, {
+            'accession':    'E-GEOD-24262',
+            'name':         'PA14_mexR vs. wildtype planktonic cells in minimal'
+                            ' medium with C-30',
+            'description':  'Mutations that made the cells insensitive to the '
+                'QS inhibition by C-30 were identified in mexR, a multi-drug '
+                'resistance operon repressor. Gene expression of mexR mutant '
+                'relative to wild-type at the presence of C-30 was examined. '
+                'Strains: PA14_mexR and wildtype. Medium: OS minimal medium + '
+                '0.1% Adenosine as carbon source. Compounds: 50 µM C-30 added '
+                'at OD600=0.25. Time: 2 hr. Temp: 37 ºC. Cell type: Planktonic '
+                'Cells.'
+        }, {
+            'accession':    'E-GEOD-17296',
+            'name':         'Transcription profiling of Pseudomonas aeruginosa '
+                            'roxSR and anr mutant strains under aerobic '
+                            'conditions',
+            'description':  'To assess the role of two redox-sensitive '
+                'transcriptional regulators, RoxSR and ANR, in Pseudomonas '
+                'aeruginosa under aerobic conditions, microarray analysis was '
+                'performed. Transcriptome profiles of roxSR mutant and anr '
+                'mutant aerobically grown in LB medium were determined by '
+                'Affymetrix GeneChip at both the exponential phase and early '
+                'stationary phase and compared to that of the wild type '
+                'strain. Experiment Overall Design: Pseudomonas aeruginosa '
+                'wild type (PAO1ut), roxSR mutant (ROX1), and anr mutant '
+                '(PAO6261) strains were cultivated aerobically in LB in '
+                'Erlenmeyer flasks, and total RNAs were extracted at both the '
+                'exponential phase (OD600 = 0.3) and early stationary phase '
+                '(OD600 = 1.4). The experiment was performed in duplicate '
+                'independent cultures.'
+        }
+    ]
+
+    def setUp(self):
+        haystack.connections.reload('default')
```

Comment:
Right... those two links came up in my search, too. It would be nice to have an authoritative solution in the form of Haystack documentation, but looking over the source code clinched it for me.